src/main/java/org/snomed/snowstorm/rest/ConceptController.java

```diff
@@ -580,6 +589,143 @@ public ExpressionStringPojo getConceptNormalForm(
+				SAxiomRepresentation axiomRepresentation = axiomConversionService.convertAxiomMemberToAxiomRepresentation(axiomMember);
+				if (axiomRepresentation != null) {
+					Set<Relationship> relationships = axiomRepresentation.getLeftHandSideNamedConcept() != null
+							? axiomRepresentation.getRightHandSideRelationships()
+							: axiomRepresentation.getLeftHandSideRelationships();
+
+					for (Relationship relationship : relationships) {
+						String targetConceptId = relationship.getDestinationId();
+						Concept targetConcept = conceptService.find(targetConceptId, sourceBranchPath);
+						if (targetConcept == null) {
+							throw new ServiceException("Axiom " + axiomMember.getId() + ": target concept " + targetConceptId + " does not exist on path " + sourceBranchPath);
+						}
+						if (includeDependencies && targetConcept.getModuleId().equals(sourceModuleId)) {
+							copyConcept(new ConceptMini(targetConcept, languageDialects), sourceBranchPath, destinationBranchPath, languageDialects, true);
+						} else if (!conceptService.exists(targetConceptId, destinationBranchPath)) {
+							throw new ServiceException("Axiom " + axiomMember.getId() + ": target concept " + targetConceptId + " does not exist on path " + destinationBranchPath);
+						}
+					}
+				}
+			} catch (ConversionException e) {
+				throw new ServiceException("Failed to deserialise axiom " + axiomMember.getId(), e);
+			}
+			referenceSetMembers.add(axiomMember);
+		}
+
+		// Create concept
+		Concept conceptVersion = new Concept(concept.getConceptId(), null, concept.getActive(), destinationModuleId, concept.getDefinitionStatus());
+
+		// Add descriptions
+		concept.getActiveDescriptions().stream()
+				.filter(description ->
+						description.isReleased() &&
+								description.getAcceptabilityMap().containsKey(Concepts.US_EN_LANG_REFSET) &&
+								description.getAcceptabilityMap().get(Concepts.US_EN_LANG_REFSET).equals(Concepts.PREFERRED_CONSTANT))
+				.forEach(description -> {
+					conceptVersion.addDescription(new Description(
+							description.getDescriptionId(),
+							null,
+							description.isActive(),
+							destinationModuleId,
+							description.getConceptId(),
+							description.getLanguageCode(),
+							description.getTypeId(),
+							description.getTerm(),
+							description.getCaseSignificanceId()));
+
+					// Collect reference set members
+					referenceSetMembers.addAll(description.getLangRefsetMembers().stream()
+							.filter(langRefsetMember ->
+									langRefsetMember.isActive() &&
+											langRefsetMember.isReleased() &&
+											langRefsetMember.getRefsetId().equals(Concepts.US_EN_LANG_REFSET))
+							.collect(Collectors.toList()));
+				});
+
+		Concept conceptCreated = conceptService.create(conceptVersion, destinationBranchPath);
+
+		// Create reference set members
+		if (!referenceSetMembers.isEmpty()) {
+			referenceSetMemberService.createMembers(
+					destinationBranchPath,
+					referenceSetMembers.stream()
```

Comment:
I think it makes the code clearer to introduce a variable rather than using stream.